controller:
  chassis_controller:
    type: chassis_controller/ChassisController
    front_left_pid: { p: 5000.0, i: 100.0, d: 5.0 }
    front_right_pid: { p: 5000.0, i: 100.0, d: 5.0 }
    back_left_pid: { p: 5000.0, i: 100.0, d: 5.0 }
    back_right_pid: { p: 5000.0, i: 100.0, d: 5.0 }
    Wheel_track: 0.400
    Wheel_base: 0.400
    ChassisSpeedMode: False
    
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
